rpc_jobs/long_running_slave.yml

```diff
@@ -0,0 +1,102 @@
+- job:
+    name: Long-Running-Slave
+    project-type: workflow
+    concurrent: true
+    properties:
+      - build-discarder:
+          num-to-keep: 30
+    parameters:
+      # See params.yml
+      - rpc_gating_params
+      - single_use_slave_params:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+          REGION: "IAD"
+          FLAVOR: general1-8
+      - rpc_maas_params:
+          RPC_MAAS_BRANCH: master
+      - string:
+          name: JENKINS_NODE_EXECUTORS
+          default: "10"
```

Comment:
Is the default 10 executors going to be used when building the long running slave? The flavor being used has quite a bit more storage and CPUs than the current CIT build node and 10 seems fairly low.